cxx-qt-gen/src/parser/property.rs

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use syn::{Ident, Type, Visibility};
+
+/// Describes a single property for a struct
+pub struct ParsedProperty {
+    /// The [syn::Ident] of the property
+    pub ident: Ident,
+    /// The [syn::Type] of the property
+    pub ty: Type,
```

Comment:
```suggestion
    pub r#type: Type,
```

Reply:
Syn calls them `ty` i've currently been following their patterns for consistency. https://docs.rs/syn/1.0.99/syn/struct.Field.html